package.json

```diff
@@ -10,7 +10,7 @@
     "cy:open": "cypress open",
     "cy:run": "cypress run",
     "dev": "lerna run --parallel dev --concurrency 8",
-    "dev:scoped": "concurrently 'live-server . --host=localhost --port=8000' 'lerna run --parallel dev --scope=@nylas/components-{availability}'",
+    "dev:scoped": "concurrently 'live-server . --host=localhost --port=8000' 'lerna run --parallel dev --scope=@nylas/components-{mailbox}'",
```

Comment:
Does this need to be changed to `mailbox`?

Reply:
No, it does not need to be. This was for my local, will remove

Reply:
For what it's worth, this is an arbitrary component name -- there's no harm in editing this to any other comopnent for running dev:scoped locally.